apps/dashboard/app/models/motd_formatter.rb

```diff
@@ -0,0 +1,140 @@
+module MotdFormatter
+  require 'rss'
+
+  class Markdown
```

Comment:
While this is still valid, I think the restructure I'm looking for is just a `motdformatter/` directory here with all these classes are still in their own file just in a sub directory.

Even more though, I think we should alias the new classes like `MotdFormatterMarkdown` -> `MotdFormatter::Markdown` so we don't even have to change the tests.  That's to keep backward comparability in the off chance that someone out there has either created their own entirely or used a rails initializer to modify one of these.

Reply:
Yeah, that was my initial approach. I'll implement that back again.